src/main/webapp/college-info.js

```diff
@@ -67,37 +67,89 @@ function loadSchoolInfo() {
-  const admissionsDiv = document.getElementById('admissions');
-  admissionsDiv.innerHTML = '';
-  admissionsDiv.append(`Acceptance Rate: ${acceptanceRate}%`);
-  admissionsDiv.append(`Average SAT Score: ${avgSat}`);
-  admissionsDiv.append(`Average ACT Score: ${avgAct}`);
-
-  // Students Section.
-  const studentsDiv = document.getElementById('students');
-  studentsDiv.innerHTML = '';
-  studentsDiv.append(`Population: ${numStudents} Students`);
-  studentsDiv.append(`4 Year Graduation Rate: ${graduationRate4yr}%`);
-
-  chartsPromise.then(() => {
-    drawRaceChart(numWhiteStudents, numAsianStudents, numBlackStudents,
-        numHispanicStudents, numIndigenousStudents, numMultiracialStudents,
-        numUnreportedRaceStudents);
-    drawGenderChart(numMen, numWomen);
-  });
+  /**
+  * Loads school description to page.
+  */
+  function getSchoolDesc() {
+    const schoolDesc = document.getElementById('school-desc');
+    const header = document.createElement('h5');
+    const para = document.createElement('p');
+    const title = document.createTextNode('About');
+    header.appendChild(title);
+    para.append(`${name} is a ${ownership} University 
+        in ${city}, ${state}`);
+    schoolDesc.appendChild(header);
+    schoolDesc.appendChild(para);
+  }
+
+  /**
+  * Loads school population and graduation rate to page.
+  */
+  function getPopulation() {
+    const population = document.getElementById('students');
+    const header = document.createElement('h5');
+    const para = document.createElement('p');
+    const title = document.createTextNode('Population and Graduation Rate');
+    header.appendChild(title);
+    para.append(`${numStudents} students, 4 Year Graduation Rate of
+        ${graduationRate4yr}%`);
+    population.appendChild(header);
+    population.appendChild(para);
+  }
+
+  /**
+  * Loads admissions statistics to page.
+  */
+  function getAdmissions() {
+    const admissions = document.getElementById('admissions');
+    const header = document.createElement('h5');
+    const para = document.createElement('p');
+    const title = document.createTextNode('Admissions');
+    header.appendChild(title);
+    para.append(`Acceptance Rate is ${acceptanceRate}%,
+        Average SAT Score is ${avgSat},
+        Average ACT Score is ${avgAct}`);
+    admissions.appendChild(header);
+    admissions.appendChild(para);
+  }
+
+  /**
+  * Loads cost of attendance to page.
+  */
+  function getCost() {
+    const cost = document.getElementById('cost');
+    const header = document.createElement('h5');
+    const para = document.createElement('p');
+    const title = document.createTextNode('Cost of Attendance');
+    header.appendChild(title);
+    para.append(`In-State Tuition is $${inStateTuition}, 
+        Out-of-State Tuition is $${outOfStateTuition}`);
+    cost.appendChild(header);
+    cost.appendChild(para);
+  }
+
+  /**
+  * Loads demographic charts to page.
+  */
+  function getCharts() {
```

Comment:
Was thinking about the refactoring Scott was talking about above and how you would call the variables to drawChart. I was originally thinking using local storage but that probably isn't necessary because you can just pass all the num variables for both graphs as parameters into getCharts. Just a though if you have not done that part already.

Reply:
I was thinking about this, because the charts don't really need to be in a separate function. The reason I made functions for everything else is to create headers and format each section. Should I just leave the charts in the loadSchoolInfo() function?